docs/insights-archive-sample/config/secrets/openshift-config/support/data.json

```diff
@@ -0,0 +1,3 @@
+{
+  "conditionalGathererEndpoint": "aHR0cDovL2xvY2FsaG9zdDo4MDgw"
```

Comment:
I don't understand why this is in a separate file. I think having one file containing the data would be clearer.

Reply:
This one contains the values of the support secret as is (with anonymization of course), the other one is the representation of IO's actual config (with anonymization again).